eng/Version.Details.xml

```diff
@@ -1,168 +1,168 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Bcl.TimeProvider" Version="8.0.0-preview.6.23280.5" CoherentParentDependency="Microsoft.AspNetCore.App.Runtime.win-x64">
```

Comment:
Why are we dropping the coherency? We don't pull the runtime packages directly, and without the coherency we won't get updates, and it'd likely create version conflicts.